app/controllers/budgets/investments_controller.rb

```diff
@@ -77,6 +77,26 @@ def suggest
       super
     end
 
+    def edit_image
+    end
+
+    def update_image
+      if @investment.update(investment_params)
+        redirect_to budget_investment_path(@investment.budget, @investment),
+                    notice: t("flash.actions.update_image.budget_investment")
+      else
+        render :edit_image
+      end
+    end
+
+    def remove_image
+      @investment.image = nil
```

Comment:
I suspected it but it wasn't clear for me with my poor knowledge of paperclip, I agree .destroy is more explicit about what's happening for other developers to easily and quickly get the output of that action 👍